### DISCUSSION TITLE
Growth Required Light Level

### DISCUSSION THREAD

**Discussion by mattisadev** [PaperMC/Paper#10587] [Ideas]
### Is your feature request related to a problem?

I would like this feature added to allow players to grow crops without having torches or another light source of 9 near.

### Describe the solution you'd like.

A light level value in the configuration file where users can specifiy the minimum light level to grow crops. Setting to 0 would practically disable the requirement of a light to grow crops.

### Describe alternatives you've considered.

The light level of 9 is hard coded into Spigot and/or paper and is not configurable. Creating a plugin to combat this is not possible as Paper is forcing the light level of 9 to be the minimum of crop growth.

**Comment by JamieIsGeek** [comment]
I know there is a config option for mob light levels so I don't see why this couldn't be added if it hasn't been done already.
